samples/sensor/bmi160/arduino_101_sss.overlay

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+&spi1 {
+	status = "ok";
+
+	bmi160@0 {
+		spi-max-frequency = <88>;
```

Comment:
Why isn't this the default setting on the board?

Reply:
No idea, I was just duplicating how the test prj.conf was setting the value.

Reply:
I think this may be a remnant of the legacy api and should be removed. The units in the old api may have been kHz (I don't remember offhand), and the sensor hardware supports up to 10 MHz. It seems crazy to run at 88 Hz.

@nashif Can you or someone at Intel please verify?

Reply:
@nniranjhana @spoorthik @kumarvikash1  can you please verify this?

Reply:
It's wrong. Actually in the same patch, in boards/arc/arduino_101_sss/arduino_101_sss.dts it gets a value of 640000 (640khz, I remember there was no possibility to use gpio as CS to overcome controller's flacky CS handling, so speed had to be quite slow).

Reply:
88Hz, no it way too slow. I am not even sure it's supported by the controller actually.
Just set 640000.